open_flamingo/scripts/run_eval.sh

```diff
@@ -4,38 +4,41 @@ eval "$(conda shell.bash hook)"
 conda activate openflamingo
 which python
 
-LM_PATH="facebook/opt-1.3b"
-LM_TOKENIZER_PATH="facebook/opt-1.3b"
-CLIP_PATH="openai/clip-vit-large-patch14"
-CKPT_PATH="/mmfs1/gscratch/efml/anasa2/latest_checkpoint.pt"
+LM_PATH="<path to llama model>"
+LM_TOKENIZER_PATH="<path to llama model>"
+VISION_ENCODER_NAME="ViT-L-14"
+VISION_ENCODER_PRETRAINED="openai"
+CKPT_PATH="/models/checkpoint.pt"
 DEVICE="0"
 
-# Default values for running on klone/hyak; change to override.
-COCO_IMG_PATH="/data/yfcc-tmp/data/mscoco/train2017/"
-COCO_ANNO_PATH="/data/yfcc-tmp/data/mscoco/annotations/captions_train2017.json"
-VQAV2_IMG_PATH="/mmfs1/gscratch/efml/anasa2/data/vqav2/train2014"
-VQAV2_ANNO_PATH="/mmfs1/gscratch/efml/anasa2/data/vqav2/v2_mscoco_train2014_annotations.json"
-VQAV2_QUESTION_PATH="/mmfs1/gscratch/efml/anasa2/data/vqav2/v2_OpenEnded_mscoco_train2014_questions.json"
-IMAGENET_ROOT="/tmp/val"
+COCO_IMG_PATH="/mscoco/train2017/"
```

Comment:
is this an actual path that will work? otherwise why not `<path to coco>/train2017`?